arch/arm64/boot/dts/qcom/msm8996.dtsi

```diff
@@ -2139,6 +2144,7 @@
 		vdd-wlan-core-supply = <&pm8994_s3>;
 		wlan-ant-switch-supply = <&pm8994_l18_pin_ctrl>;
 		qcom,wlan-en-vreg-support;
+		qcom,enable-bootstrap-gpio;
```

Comment:
This one is weird
k3.18 version: e22e8f5f77a93a06ae8383f60009864e040b9a00
k4.4 version: 45bdc1b602c5c25498b1adbae6f9e0ecc540575f